lib/ignore/ignore.go

```diff
@@ -406,11 +406,19 @@ func loadParseIncludeFile(filesystem fs.Filesystem, file string, cd ChangeDetect
 	}
 
 	if cd.Seen(filesystem, file) {
-		return nil, parseError(fmt.Errorf("multiple include of ignore file %q", file))
+		return nil, errors.New("multiple include")
```

Comment:
Why shouldn't this be a parse error?

Reply:
For the same reason as why there was no need to put the initially changed error inside a parseError, the returning error from this function is always being wrapped inside a parseError already;
`err = parseError(fmt.Errorf("failed to load include file %s: %w", includeFile, err)) `. I don't think it adds much at this point?

Reply:
Thanks. (Damn I am thinking slowly (to the point of not thinking) today).